hack/preview.sh

```diff
@@ -86,3 +97,26 @@ git checkout $MY_GIT_BRANCH
 
 #set the local cluster to point to the current git repo and branch and update the path to development
 $ROOT/hack/util-update-app-of-apps.sh $MY_GIT_REPO_URL development $PREVIEW_BRANCH
+
+# Make sure we have a tekton-chains namespace
+echo "Checking to see if tekton-chains namespace exists"
+while ! kubectl get namespace tekton-chains &> /dev/null; do
+  echo -n .
+  sleep 3
+done
+
+echo "Setting chains to use cluster rekor server: https://rekor.$domain"
+kubectl patch configmap/chains-config -n tekton-chains --patch "{\"data\":{\"transparency.url\":\"https://rekor.$domain\"}}" --type=merge 
+# Delete the controller pod for chains to ensure that the configuration change gets picked up.
+echo "Restarting chains controller"
+kubectl delete pod -n tekton-chains -l app=tekton-chains-controller
+# If we have a rekor namespace, we should wait for the server to be available
+if kubectl get namespace &>/dev/null; then
```

Comment:
Not sure what the intention is here. Was it supposed to be `kubectl get namespace enterprise-contract-service` ?